scripts/halium

```diff
@@ -57,6 +57,18 @@ identify_boot_mode() {
 		esac
 	fi
 
+	# On Android 8+ devices the 'android' boot mode is broken and should be avoided.
+	# This behavior can be overridden with the cmdline flag 'halium_no_avoid_android_mode'
+	if ! grep -q halium_no_avoid_android_mode /proc/cmdline; then
```

Comment:
Maybe make sure it's a full word?

```suggestion
	if ! grep -w -q halium_no_avoid_android_mode /proc/cmdline; then
```

Reply:
I suppose `grep -wq` would be fine as well?

Reply:
Probably. Please test.

Reply:
`grep -wq` works just fine as well :+1: will update this PR in a moment